mlxbf-bootctl.c

```diff
@@ -1310,18 +1393,24 @@ int main(int argc, char **argv)
     "                      [--output|-o OUTPUT] [--read|-r INPUT]\n"
     "                      [--bootstream|-b BFBFILE] [--overwrite-current]\n"
     "                      [--watchdog-swap interval | --nowatchdog-swap]\n"
+    "                      [--watchdog-boot-mode disabled|standard|time_limit]\n"
+    "                      [--watchdog-boot-interval interval]\n"
```

Comment:
Question: would it be possible to specify the interval? e.g., 45..4095

Reply:
Done